model.js

```diff
@@ -1,7 +1,18 @@
-const db = require("./db/connection")
+const db = require("./db/connection");
+const {
+  convertTimestampToDate,
+  createRef,
+  formatComments,
+} = require("./db/seeds/utils");
 
 fetchCategories = () => {
-return db.query(`SELECT * FROM categories;`)
-}
+  return db.query(`SELECT * FROM categories;`);
+};
 
-module.exports = { fetchCategories }
\ No newline at end of file
+fetchReviews = () => {
+  //   let date = convertTimestampToDate({ created_at: 1610964101251 });
+  //   console.log(date);
+  return db.query(`SELECT * FROM reviews`);

```

Comment:
combine aggregate function COUNT with AS in your query to get back what you want. 
It's in the NC notes on SQL